packages/docker/data_stream/cpu/fields/fields.yml

```
@@ -3,6 +3,10 @@
   release: ga
   description: |
     Container labels
+- name: container.cpu.usage
+  type: scaled_float
+  format: percent
+  description: ECS field for container CPU usage
```

You can import definitions of ECS fields with `external: ecs`. Consider doing this at least for all the ECS fields added in this PR.
```suggestion
- name: container.cpu.usage
  external: ecs
```

ECS version used is defined in `_dev/build/build.yml`.